dataservice/api/common/schemas.py

```diff
@@ -12,6 +12,7 @@ def __init__(self, code=200, message='success', *args, **kwargs):
 
     class Meta:
         exclude = ('_id', 'uuid')
+        dump_only = ('created_at', 'modified_at')
 
     @post_dump(pass_many=True)
     def wrap_envelope(self, data, many):
```

Comment:
Just curious, won't the many flag tell you if data is a list or not?

Reply:
The `pass_many` in the decorator makes it pass the raw data and sets the `many` arg to `True`. Otherwise, it will pass each object independently, and I don't want that because I want all entities to be wrapped in the envelope.

Reply:
Maybe I'm misunderstanding `pass_many`? But I thought `pass_many` says that the arg `data` could be a single item or many, and if it is many, then the `many` flag will be set to true. Then you would just check `if not many:` instead of `if type(data) is not list:`. I was playing around with this before and it seemed to work

Reply:
Yeah, that could be done too, just need to make sure it's always specified as `many` when it's being dumped.

Reply:
Updated to check many and also check for presence of `_links`